files/image_config/logrotate/templates/logrotate-syslog.j2

```diff
@@ -0,0 +1,29 @@
+{% from "logrotate-common.j2" import postrotate with context %}
+{% from "logrotate-common.j2"
+    import disk_percentage_treshold_rotation with context -%}
+{% from "defaults.j2" import def_logging -%}
+
+{% set syslog = (LOGGING | d({})).get('syslog', {}) -%}
+{% set frequency = syslog.frequency | d(def_logging.common.frequency) -%}
+{% set max_number = syslog.max_number | d(def_logging.syslog.max_number) -%}
+
+{% if syslog.disk_percentage and not syslog.size -%}
```

Comment:
What does disk percentage signify? Does it mean what is the max size of a syslog file in percentage of the disk?

Reply:
Right, max size of a syslog file in percentage of the disk